inheritance/src/main/java/inheritance/Restaurant.java

```diff
@@ -0,0 +1,41 @@
+package inheritance;
+
+import java.util.LinkedList;
+
+public class Restaurant implements Reviewable{
+
+    String restaurantName;
+    int stars;
+    String price;
+    LinkedList<Review> restaurantReviews;
+
+//    restaurant constructor
+    public Restaurant (String restaurantName, int stars, String price){
+
+        this.restaurantName = restaurantName;
+        this.stars = stars;
+        this.price = price;
+        this.restaurantReviews = new LinkedList<>();
+    }
+
+    @Override
+    public void addReview(Review review) {
+//        addReview adds a review to the LL using .add builtin method
+//        addReview is going to add pointer (which helps the communication between interface)
+
+        restaurantReviews.add(review);
+//        ?????
+        review.business = this;
+
+    }
+
+//    Do star rating for restaurant from LL an average & make sure it updates 
```

Comment:
Consider writing a method here that serves (1) to make a new review, and then calls your method addReview so it can then add that review to your list of reviews, and (2) update the stars with the average rating (produced from adding up all the stars in the reviews, and dividing by the number of reviews). This should solve your testing problem, because you test this new method, which takes in addReview (instead of testing addReview, directly).